ui/apps/dashboard/src/app/(organization-active)/support/statusPage.ts

```diff
@@ -1,43 +1,13 @@
 'use client';
 
 import { useEffect, useState } from 'react';
-import colors from 'tailwindcss/colors';
 
-type Indicator = 'none' | 'minor' | 'major' | 'critical';
-type StatusPageStatusResponse = {
-  page: {
-    id: string;
-    name: string;
-    url: string;
-    updated_at: string;
-  };
-  status: {
-    description: string;
-    indicator: Indicator;
-  };
-};
-
-type Status = {
-  url: string;
-  description: string;
-  indicator: Indicator;
-  indicatorColor: string;
-  updated_at: string;
-};
-
-// We use hex colors b/c tailwind only includes what is initially rendered
-export const indicatorColor: { [K in Indicator]: string } = {
-  none: colors.green['500'],
-  minor: colors.yellow['300'],
-  major: colors.orange['500'],
-  critical: colors.red['600'],
-};
-
-const STATUS_PAGE_URL = 'https://status.inngest.com';
-
-const fetchStatus = async (): Promise<StatusPageStatusResponse> => {
-  return await fetch('https://inngest.statuspage.io/api/v2/status.json').then((r) => r.json());
-};
+import {
```

Comment:
All nav items that required client side data fetching have been refactored slightly to allow server side fetching and rendering. Here I broke out the fetch and status mapping parts so they could be called from server code. I kept this hook here as it is used elsewhere.